sdk/ai/azure-ai-inference/dev_requirements.txt

```diff
@@ -3,4 +3,8 @@
 ../../core/azure-core-tracing-opentelemetry
 ../../monitor/azure-monitor-opentelemetry
 aiohttp
-opentelemetry-sdk
\ No newline at end of file
+opentelemetry-sdk
+aiofiles
```

Comment:
dev_requirements.txt is used by our CI to specify additional requirements an SDK developer might need to test the package and its samples, but these don't get installed when someone pip installs the library. It looks like we directly use `aiofiles` in the prompts src code which makes it a dependency. Additional dependencies need to be added in the setup.py and approved by our architects.

Reply:
`aiofiles` as far as I know is not on the approved list. Was it okayed to use?